Tests/Functional/RequestBodyParamConverterTest.php

```diff
@@ -28,4 +28,16 @@ public function testRequestBodyIsDeserialized()
         $this->assertSame(200, $client->getResponse()->getStatusCode());
         $this->assertSame('Post 1', $client->getResponse()->getContent());
     }
+
+    /**
+     * @see https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1237
+     */
+    public function testTwigErrorPage()
+    {
+        $client = $this->createClient(['test_case' => 'RequestBodyParamConverter']);
+        $client->request('GET', '/_error/404.txt');
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
```

Comment:
Ah, you are right. Thanks. Maybe we should add a comment to avoid confusion for anybody else stumbling over this.